.eslintrc

```diff
@@ -7,8 +7,17 @@
     "plugin:jsx-a11y/recommended"
   ],
   "plugins": ["jsx-a11y"],
+  "ignorePatterns": ["*.test.js"],
```

Comment:
Qual o motivo para ignorar o lint em arquivos de teste? Nos projetos que trabalho, os arquivos de teste tem um lint diferente por conta de algumas regras como mencionei [aqui](https://github.com/filipedeschamps/tabnews.com.br/pull/404#pullrequestreview-1040602352), mas não é desabilitado.

Reply:
Não sei se a motivação foi essa, mas no PR #404 o @filipedeschamps perguntou se dava para ignorar os testes.

Eu acho que vale a pena deixar habilitado para testes também, mas, para não travar esse PR, se o @issdomingoss não puder fazer as adequações, podemos adequar os testes em outro PR na sequência.

Obrigado pelo força @Rafatcb!

Reply:
O que acham de fazer diferente do que o que foi feito no #404?

Ao invés disso:

```js
// eslint-disable-next-line no-unused-vars
const childBranchBLevel2Content3 = await orchestrator.createContent({
```

Fazer assim:
```js
// childBranchBLevel2Content3
await orchestrator.createContent({
```

Reply:
> O que acham de fazer diferente do que o que foi feito no #404?
> 
> Ao invés disso:
> 
> ```js
> // eslint-disable-next-line no-unused-vars
> const childBranchBLevel2Content3 = await orchestrator.createContent({
> ```
> 
> Fazer assim:
> 
> ```js
> // childBranchBLevel2Content3
> await orchestrator.createContent({
> ```

Faz sentido para mim.